src/analyzer.ts

```diff
@@ -117,6 +122,18 @@ const DEFAULT_LOCKFILES: Record<Agent, string[]> = {
   pnpm: ['pnpm-lock.yaml'],
 };
 
+function isValidWireitScriptCommand(command: string): boolean {
+  return (
+    command === 'wireit' ||
+    command === 'yarn run -TB wireit' ||
+    // This form is useful when using package managers like yarn or pnpm which
+    // do not automatically add all parent directory `node_modules/.bin`
+    // folders to PATH.
+    /^(\.\.\/)+node_modules\/\.bin\/wireit$/.test(command) ||
```

Comment:
Do you ever need to handle `./node_moduules/...`? Like in a non-monorepo with Yarn>

Reply:
I think if it's installed to `./node_modules/.bin` then every package manager should work with just `wireit`, no need to write `./node_modules/...`. npm also automatically checks all parent directory `node_modules/.bin`, but yarn and pnpm don't do that. So this just handles that gap.